editor.planx.uk/src/components/Feedback/FeedbackPhaseBanner.test.tsx

```diff
@@ -0,0 +1,67 @@
+import "@testing-library/jest-dom/extend-expect";
+
+import { fireEvent } from "@testing-library/react";
```

Comment:
This isn't something we're currently enforcing (maybe we should?) but `fireEvent` is rarely needed and we can rely on `user` for an experience which is closer to the real DOM. 

Instead of importing this we can can use the `user` returned from `setup()` like so - 

```ts
test("clicking on feedback link calls handleFeedbackClick", async () => {
  const { getByText, user } = setup(
    <FeedbackPhaseBanner
      handleFeedbackClick={handleFeedbackClick}
      handleReportAnIssueClick={handleReportAnIssueClick}
    />,
  );

  await user.click(getByText("feedback"));
  expect(handleFeedbackClick).toHaveBeenCalledTimes(1);
});
```

The notable change here is that this is now an `async` function - I think the test will just fail without the `await` before `user.click()`.

Docs: https://testing-library.com/docs/dom-testing-library/api-events/

Reply:
Fixed here: 2035632e5c109b90ebf6e065192a143a7169d2c6